src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.h

```diff
@@ -534,6 +545,67 @@ namespace xclcpuemhal2 {
     static unsigned int mGraphHandle;
   };
   extern std::map<unsigned int, CpuemShim*> devices;
+ 
+  /**
+   * sParseLog structure parses a file named mFileName and looks for a matchString
+   * On successfull match, print the line to the console
+   * Currently, we are using this structure to parse the external IO file that is
+   * generated by the deviceProcess during SW EMU.
+   */
```

Comment:
```suggestion
   // sParseLog structure parses a file named mFileName and looks for a matchString
   // On successfull match, print the line to the console
   // Currently, we are using this structure to parse the external IO file that is
   // generated by the deviceProcess during SW EMU.
```
Better to use // for commenting C++ compiled code, and it is at least consistent with other comments added in this file.